packages/yoastseo/src/languageProcessing/helpers/sentence/SentenceTokenizer.js

```diff
@@ -391,6 +391,31 @@ export default class SentenceTokenizer {
 		return currentSentence.endsWith( lastAbbreviation );
 	}
 
+	/**
+	 * Checks whether the given tokens are a valid html tag pair.
+	 * Note that this method is not a full html tag validator. It should be replaced with a better solution once the html parser is implemented.
+	 * @param {string} firstToken An opening html tag.
+	 * @param {string} lastToken A closing html tag.
+	 * @returns {boolean} True if the tokens are a valid html tag pair. Otherwise, False.
+	 */
+	isValidTagPair( firstToken, lastToken ) {
+		const firstTokenText = firstToken.src;
+		const lastTokenText = lastToken.src;
+
+		// regex to get the tag type
+		const tagTypeRegex = /<\/?([^\s]+?)(\s|>)/;
```

Comment:
Question for CR: would it be worth it to make `tagTypeRegex` this a global variable?